src/widgets/helper/ChannelView.cpp

```diff
@@ -339,9 +359,26 @@ void ChannelView::themeChangedEvent()
 {
     BaseWidget::themeChangedEvent();
 
+    this->setupHighlightAnimationColors();
     this->queueLayout();
 }
 
+void ChannelView::setupHighlightAnimationColors()
+{
+    constexpr int maxOpacity = 110;
+    if (this->theme->isLightTheme())
+    {
+        this->highlightAnimation_.setStartValue(QColor(20, 20, 20, maxOpacity));
+        this->highlightAnimation_.setEndValue(QColor(20, 20, 20, 0));
+    }
+    else
+    {
+        this->highlightAnimation_.setStartValue(
+            QColor(230, 230, 230, maxOpacity));
+        this->highlightAnimation_.setEndValue(QColor(230, 230, 230, 0));
+    }
```

Comment:
Instead of checking for light theme, define two colors (`highlightAnimationStart` and `highlightAnimationEnd`) in the theme class that does the light theme checking

That way this class would not need a branch, and just reference the two new colors

Reply:
Done in aed9164c0b420ce7c937f21507db91e3bf7cef5f.